internal/cmd/tunnel.go

```diff
@@ -108,9 +109,9 @@ type client struct {
 }
 
 func (c *client) start() error {
-	url := fmt.Sprintf("%s%s%s%s%s", c.brokerAddr, "/api/private/envagent/", c.id, "/connect?session_token=", c.token)
-	c.logger.Info(c.ctx, "connecting to broker", slog.F("url", url))
-
+	url := fmt.Sprintf("%s%s%s%s%s", c.brokerAddr.String(), "/api/private/envagent/", c.id, "/connect?session_token=", c.token)
+	tcpProxy := fmt.Sprintf("turns:%s:5349?transport=tcp", c.brokerAddr.Host)
```

Comment:
`turns` is using tls right? We could technically look at the broker addr scheme here and decide if it's TURN or TURNs.

Reply:
Good point... I'll change this now!